src/vmm/src/device_manager/persist.rs

```diff
@@ -420,23 +419,25 @@ mod tests {
                 network_interface,
             );
             // Add a vsock device.
-            let orig_tmp_sock_file = TempSockFile::new(TempFile::new().unwrap());
+            let mut orig_tmp_sock_file = TempFile::new().unwrap();
+            orig_tmp_sock_file.remove();
```

Comment:
You don't need a new `TempFile`, you can use the one above.
```suggestion
```

Reply:
Removed `orig_tmp_sock_file` and updated `VsockDeviceConfig` to use `tmp_sock_file`